utilities/cli/container.py

```
@@ -772,75 +793,128 @@ def get_environment_args(self) -> List[str]:
-                                ["xauth", "-f", xauth_file, "nmerge", tmp.name],
-                                check=False,
-                                dry_run=self.dryrun,
-                            )
-                        options.extend(
-                            ["-v", f"{xauth_file}:{xauth_file}", "-e", f"XAUTHORITY={xauth_file}"]
-                        )
+        if wayland_display:
+            options.extend(["-e", "WAYLAND_DISPLAY"])
+
+        if display:
+            if Path("/tmp/.X11-unix").is_dir() and not self._is_ssh_x11_display(display):
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/.X11-unix"

(S108)